static/js/gtag.js

```diff
@@ -1,4 +1,39 @@
 window.dataLayer = window.dataLayer || [];
-function gtag(){dataLayer.push(arguments);}
+function gtag() { dataLayer.push(arguments); }
 gtag('js', new Date());
 gtag('config', 'UA-26406144-13');
+
+// Setup listening to send Google Analytics events when any element
+// with a 'data-ga-click-event' attribute is clicked.
+window.addEventListener('DOMContentLoaded', function () {
+  function findPackageName() {
+    var pageData = document.getElementsByName('pub-page-data')[0];
+    if (pageData) {
+      var pkgData = JSON.parse(atob(pageData.content)).pkgData || {};
+      var pkg = pkgData.package;
+      if (!pkg) {
+        console.log('Failed to identify package name from pageData');
+        return null;
+      }
+      return 'package:' + pkg;
+    }
+    return null;
+  }
+  var label = findPackageName() || '-';
+
+  function sendEvent(e) {
+    gtag('event', e.currentTarget.dataset.gaClickEvent, {
+      'event_category': 'click',
+      'event_label': label,
```

Comment:
Can we have a 'package' parameter instead? Or let's add `package:` prefix to the value, so publishers can also be tracked similarly.

Reply:
We already have a package prefix added in `findPackageName`.

As I understood events: category is for sorting... action is the thing we count, and label (optional) is a bit of extra context..

On a given session an action can ONLY be counted once.. so label shouldn't necessarily be thought of as an extra level...

Okay, maybe I only somewhat understand the docs... :)

Reply:
Maybe I should move the prefixing, so that it's less obscure... Or what are you suggesting..

Reply:
Maybe we should emit the value that we want to have here in the main HTML, so no lookup is needed?

Reply:
My thinking was to use the page `<title>` at one point... or maybe the path from the URL, but stripping away query string and hashes...